### ISSUE TITLE
Can't Resolve tns-core-modules/utils/utils.ios

### ISSUE THREAD

**Issue by AngeloAnolin** [triniwiz/nativescript-stripe#87]
TNS Version: 6.2.0

Steps to Reproduce:
1. Create a new project (Vue) with the command:
`vue init nativescript-vue/vue-cli-template <project-name>`

2. Go to the project folder and run the following command:
```
cd <project-name>
npm install
```

At this stage, you can check if the app would run through the command:
`tns run ios --bundle`
or
`tns run android`

4. Add the plugin:
`tns plugin add nativescript-stripe`

5. Modify main.js so that the `<CreditCardView />` component can be used.
`Vue.registerElement('CreditCardView', () => require('nativescript-stripe').CreditCardView)`

or, Simply open the App.vue component and import the Stripe component:

```
<script>
import { Stripe, Card } from 'nativescript-stripe'
export default {
  .....
}
</script>
```
6. Try to run the application and you will see the errors below:

```
ERROR in ../node_modules/nativescript-stripe/stripe.js
Module not found: Error: Can't resolve 'tns-core-modules/utils/utils.ios' in '/Users/angelo/Projects/stripesimple/node_modules/nativescript-stripe'
 @ ../node_modules/nativescript-stripe/stripe.js 11:18-61
 @ ./main.js
```

I think this might be related to the new NS version that I've updated my NS installation with.

**Comment by RobertGardner** [comment]
I upgraded to NS 6.2 and had a different but possibly related problem. I suspect it's a webpack problem. I've filed [this bug](https://github.com/NativeScript/nativescript-dev-webpack/issues/1094) with NativeScript.

**Comment by AngeloAnolin** [comment]
For those who are looking for a quick fix to make this work.

1. Locate the file `stripe.ios.js` under the `node_modules\nativescript-stripe` folder.
2. Replace the line of code (line 5) `var utils_ios_1 = require("tns-core-modules/utils/utils.ios");` with `var utils_ios_1 = require("tns-core-modules/utils/utils");`
3. Perform a platform remove command (i.e. `rm -rf platforms`).
4. Rebuild your application.

**Comment by Sauvetonbio** [comment]
Hello, can you please fix it in next version ? We use cloud builds, so we can't update node_modules files.

Thank you !

**Comment by RobertGardner** [comment]
Fixed in v6.2.1.
